### DISCUSSION TITLE
Bienvenido a las discusiones de Python-Para-Principiantes!

### DISCUSSION THREAD

**Discussion by danielgara** [PracticalBooks/Python-Para-Principiantes#1] [Announcements]
## 👋 Bienvenido!
  Utilizaremos este lugar para conectarnos con otros lectores y miembros de la comunidad. Esperamos que:
  * Realices preguntas sobre el libro o programación en general.
  * Compartas ideas.
  * Te relaciones con otros lectores o miembros de la comunidad.
  * Le des la bienvenida a otros.

  Si lo deseas, podrías dejar una pequeña descripción sobre quién eres y cómo llegaste a este espacio.

**Comment by jonnyhdz13** [comment]
Hola Grupo!,

no he logrado encontrar la solución del ejercicio E10.6. Un filólogo (persona que estudia las lenguas) está realizando una investigación para comunicarse con alienígenas. Todos sabemos que los alienígenas no utilizan vocales. Por lo tanto, el filólogo desea contar con un programa, que, al ingresarle una palabra, muestre cada letra de esa palabra en una línea independiente. Y si alguna letra corresponde a una vocal, no la muestre. Implemente un programa en el cual le solicite al usuario que ingrese por teclado una palabra. Luego, imprima por pantalla cada carácter de la palabra en una línea independiente (sin incluir los caracteres que representan vocales).

me pueden ayudar por favor..

**Reply by danielgara** [reply]
Hola @jonnyhdz13 en este enlace encuentras la solución a todos los ejercicios: https://github.com/PracticalBooks/Python-Para-Principiantes/blob/main/Capitulo19/Cap-19-Solucion-de-ejercicios.ipynb -> búscalo por E10.6. Un saludo.

**Comment by jonnyhdz13** [comment]
Hola Daniel!

Super, ya con esto comprendí el problema, estoy muy contento con este
libro. espero avanzar bastante.

Muchas gracias,

**Comment by fjaramillp** [comment]
Hola grupo,                                                                                                                                                                                                   
estoy teniendo unos inconvenientes con el capítulo de diccionarios al intentar imprimir los diccionarios usando print(producto) me sale un error, revise la estructura, pero la verdad no veo el error, alguna sugerencia?     
![image](https://user-images.githubusercontent.com/126544339/224826772-731b69c3-aea3-43dc-b2ad-05b8fcf88968.png)

**Reply by danielgara** [reply]
Hola @fjaramillp intenta siguiendo la recomendación de la siguiente imagen. 

![image](https://user-images.githubusercontent.com/2061370/224828813-0a7d736f-25f5-4b4f-91a3-100c20ce68a6.png)

**Reply by fjaramillp** [reply]
Listo Daniel, muchísimas gracias ya mismo lo voy a intentar

**Reply by JessEmmanuel34** [reply]
Esa explicación esta en el capitulo 4 pagina 31.
